src/browser-connection/gateway.js

```diff
@@ -36,7 +37,8 @@ export default class BrowserConnectionGateway {
         this._dispatch('/browser/status/{id}', proxy, BrowserConnectionGateway.onStatusRequest);
 
         proxy.GET('/browser/assets/index.js', { content: IDLE_PAGE_SCRIPT, contentType: 'application/x-javascript' });
-        proxy.GET('/browser/assets/style.css', { content: IDLE_PAGE_STYLE, contentType: 'text/css' });
+        proxy.GET('/browser/assets/styles.css', { content: IDLE_PAGE_STYLE, contentType: 'text/css' });
+        proxy.GET('/favicon.png', { content: FAVICON, contentType: 'image/x-icon' });
```

Comment:
I'll create a separate issue for this in the [testcafe-hammerhead](https://github.com/superroma/testcafe-hammerhead) repo.
@helen-dikareva Let's move `favicon.ico` registering to the `TestCafe` class (as @inikulin  wrote). It'll work when we update the `testcafe-hammerhead` version